modules/nf-core/argnorm/meta.yml

```diff
@@ -0,0 +1,65 @@
+name: "argnorm"
+description: Normalize antibiotic resistance genes (ARGs) using the ARO ontology (developed by CARD).
+keywords:
+  - amr
+  - antimicrobial resistance
+  - arg
+  - antimicrobial resistance genes
+  - genomics
+  - metagenomics
+  - normalization
+  - drug categorization
+tools:
+  - "argnorm":
+      description: "Normalize antibiotic resistance genes (ARGs) using the ARO ontology (developed by CARD)."
+      homepage: "https://argnorm.readthedocs.io/en/latest/"
+      documentation: "https://argnorm.readthedocs.io/en/latest/"
+      tool_dev_url: "https://github.com/BigDataBiology/argNorm"
+      licence: ["MIT"]
+
+input:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. `[ id:'sample1', single_end:false ]`
+
+  - input_tsv:
+      type: file
+      description: ARG annotation output
+      pattern: "*.tsv"
+
+  - db:
+      type: string
+      description: Database used for ARG annotation
+      pattern: "sarg|ncbi|resfinder|deeparg|megares|argannot|resfinderfg"
+
+  - hamronized:
+      type: string
+      description: Indicate ARG annotation output has been processed by hAMRonization
+      pattern: "|--hamronized"
```

Comment:
This should be true/false, and the code change the command appropriately

Reply:
I've just updated the code now. Is it ok?

Reply:
Hi @Vedanth-Ramji . Can you also update `main.nf` with something like https://github.com/nf-core/modules/blob/master/modules/nf-core/khmer/normalizebymedian/main.nf#L24 ?